static/scss/dashboard.scss

```diff
@@ -47,6 +47,12 @@ c.validation-wrapper {
       }
     }
 
+    .bottom-align-text {
```

Comment:
you added this class, but i can't find any code in this diff where it's being used. why was this added?